packages/admin/admin/src/tabs/RouterTabs.tsx

```diff
@@ -124,20 +124,23 @@ function RouterTabsComponent({
                 return (
                     <Route path={path}>
                         {({ match }) => {
-                            if (match && stackApi && stackSwitchApi && !foundFirstMatch) {
+                            let ret = null;
+                            if (match && !foundFirstMatch) {
                                 foundFirstMatch = true;
+                                ret = <div className={classes.content}>{child.props.children}</div>;
+                            } else if (child.props.forceRender) {
+                                ret = <div className={`${classes.content} ${classes.contentHidden}`}>{child.props.children}</div>;
+                            } else {
+                                return null;
+                            }
```

Comment:
I don't think this early return is correct. The `if (stackApi && stackSwitchApi)` will never be executed

Reply:
I added a comment that explains why the if doesn't have to be executed